python/pip_install/pip_repository.bzl

```diff
@@ -699,7 +708,7 @@ def _whl_library_impl(rctx):
     )
 
     if result.return_code:
-        fail("whl_library %s failed: %s (%s)" % (rctx.attr.name, result.stdout, result.stderr))
+        fail("whl_library %s failed: %s (%s) error code: '%s'" % (rctx.attr.name, result.stdout, result.stderr, result.return_code))
```

Comment:
Since we're here...

```suggestion
        fail(("whl_library '{name}' failed: wheel_installer failed\n" +
              "command: {args}\n" +
              "error code: {error_code}\n" +
              "===== stdout ====={stdout}\n" +
              "===== stderr ====={stderr}\n=====").format(
              name = rctx.attr.name,
              command = args,
              code=result.return_code,
              stdout=result.stdout,
              stderr=result.stderr
            ))
```

Reply:
Did you test this code?

Reply:
no. i'm fine if you want to skip it for now.